lib/tasks/spotlight.rake

```diff
@@ -10,4 +10,18 @@ namespace :spotlight do
       system 'bundle exec rake db:migrate'
     end
   end
+  desc 'Remove empty anchors'
+  task remediate_anchors: :environment do
+    url_regex = %r{<a href="(https?|mailto:)[\S]+">(<br>)?<\/a>}
```

Comment:
This matches all of the examples that were posted but might not handle all cases